pom.xml

```diff
@@ -102,7 +103,8 @@
     <version.org.simpleframework>6.0.1</version.org.simpleframework>
     <!-- this version will overwrite jboss-ip-bom version 2.5.1. The version in jboss-ip-bom couldn't be upgraded as it will break ModeShape Web Explorer application which uses GWT 2.5.1 -->
     <!-- please look at https://github.com/fredsa/gwt-dnd/wiki/GettingStarted -->
-    <version.com.google.gwt>2.7.0</version.com.google.gwt>
+    <version.com.google.gwt>2.8.0-beta1</version.com.google.gwt>
+    <version.com.google.guava>20.0-SNAPSHOT</version.com.google.guava>
```

Comment:
I also really dislike this, but unfortunately there is no released version of guava that is compatible with GWT 2.8. Using guava 19 (the most recent version) causes build errors. There is literally no stable version of guava we can use.